src/table/src/test_util.rs

```diff
@@ -19,3 +19,39 @@ mod mock_engine;
 pub use empty_table::EmptyTable;
 pub use memtable::MemTable;
 pub use mock_engine::MockTableEngine;
+
+#[inline]
+fn region_name(table_id: u32, n: u32) -> String {
+    format!("{table_id}_{n:010}")
+}
+
+#[inline]
+fn table_dir(catalog_name: &str, schema_name: &str, table_id: u32) -> String {
+    format!("{catalog_name}/{schema_name}/{table_id}")
+}
```

Comment:
They are duplicated with the methods under `mito::engine`. I think we could define them as common methods for `mito` and `table` crate.

Reply:
Yep, I'm looking for a suitable crate to place these methods. But I didn't find a good one for me. Would you happen to have any suggestions?

Reply:
For now, I re-export these methods from `mito`, and import them in the datanode.

Reply:
Well, what about putting them in the `table` crate. We should share the same rules to construct the table_dir/region_name/region_id

Reply:
> Well, what about putting them in the `table` crate. We should share the same rules to construct the table_dir/region_name/region_id

Well, I will move them later.